lib/Jsrt/Jsrt.cpp

```diff
@@ -2746,3 +2762,47 @@ CHAKRA_API JsRunSerializedScriptWithCallback(_In_ JsSerializedScriptLoadSourceCa
     return RunSerializedScriptCore<JsSerializedScriptLoadSourceCallback, JsSerializedScriptUnloadCallback>(nullptr, scriptLoadCallback, scriptUnloadCallback, buffer, sourceContext, sourceUrl, false, result);
 }
 #endif // _WIN32
+
+
+CHAKRA_API JsRunScriptUtf8(
```

Comment:
Agreed, I thought of the same. There are other aspects of the APIs we may need to iterate on, e.g. lifetime contract of the script source.